collect_app/src/main/java/org/odk/collect/android/external/ExternalSQLiteOpenHelper.java

```diff
@@ -133,9 +133,11 @@ private void onCreateNamed(SQLiteDatabase db, String tableName) throws Exception
 
             boolean sortColumnAlreadyPresent = false;
 
-            sb.append("CREATE TABLE ");
-            sb.append(tableName);
-            sb.append(" ( ");
+            sb
+                    .append("CREATE TABLE ")
+                    .append(tableName)
+                    .append(" ( ");
+
```

Comment:
I’d go further and make it like this, but I realize that’s not this PR’s purpose.

![screen shot 2018-06-28 at 11 30 05 am](https://user-images.githubusercontent.com/55099/42053541-a8b8a46a-7ac6-11e8-9a9a-ecbb3cb1b170.png)